### ISSUE TITLE
Add print label link to order list

### ISSUE THREAD

**Issue by v18** [Automattic/woocommerce-services#962]
In our user tests, users would look for ways to print a label directly from the orders list page:

yourstore.com/wp-admin/edit.php?post_type=shop_order

<img width="1506" alt="screen shot 2017-03-28 at 11 25 02 pm" src="https://cloud.githubusercontent.com/assets/11487924/24459945/d359ecc4-146a-11e7-8e17-754e39ecb2d8.png">

#user-test-march-2017

**Comment by jeffstieler** [comment]
Duplicate of https://github.com/Automattic/woocommerce-services/issues/601

Wherever the work gets done, let's make sure to close both issues when complete.

(cc: @olaolusoga)

**Comment by jeffstieler** [comment]
👋 @kellychoffman 

A few things:
* I think this should happen after https://github.com/Automattic/woocommerce-services/issues/1117 and https://github.com/Automattic/woocommerce-services/issues/819 _(at least the "complete order" part)_
* Should this open the label modal on this screen, or kick people over to the order detail screen with the modal open?
* What icon makes sense for "print a shipping label"? Does context help? _(put in 'ship to' column instead of 'actions')

**Comment by kellychoffman** [comment]
> I think this should happen after #1117 and #819 (at least the "complete order" part)

Agree.

> Should this open the label modal on this screen, or kick people over to the order detail screen with the modal open?

Since the label modal is pretty comprehensive and includes customer info, products in order, etc I think its okay to keep people in the list screen. That way they can print another one quickly and easily. 

> What icon makes sense for "print a shipping label"? Does context help? _(put in 'ship to' column instead of 'actions')

This is tricky because icon-only action buttons are not ideal. We should really rethink that across the board for WC, but in the meantime, let's use a 🖨 icon and have the tooltip text be 'Print shipping label'. Extensions on WC have a bad habit of just adding another column to these list views, which leads to an unreadable and barely usable page.

**Comment by dechov** [comment]
There is no longer either a "Ship to" or an "Actions" column in this view, as of [WooCommerce 3.3](https://woocommerce.com/2018/01/whats-new-woocommerce-3-3/). Is that an obstacle here, or would adding a new column be warranted?

**Comment by laurendavissmith** [comment]
Closing since we closed #601 and this is a dupe of that one
